aerogear-bom/pom.xml

```diff
@@ -363,13 +363,7 @@
 
             <dependency>
               <groupId>org.keycloak</groupId>
-              <artifactId>keycloak-model-sessions-mem</artifactId>
-              <version>${keycloak.version}</version>
-            </dependency>
-
-            <dependency>
-              <groupId>org.keycloak</groupId>
-              <artifactId>keycloak-model-sessions-jpa</artifactId>
+              <artifactId>keycloak-model-sessions-infinispan</artifactId>
               <version>${keycloak.version}</version>
```

Comment:
@matzew are we going to use UPS in a clustered environment? If yes, that's required, otherwise, I don't think so.

Take a look: http://keycloak.github.io/docs/userguide/keycloak-server/html/clustering.html